src/templates/LDCETemplate.ts

```diff
@@ -0,0 +1,205 @@
+import { getString as _getString, getBoolean as _getBoolean } from "./helpers";
+import { buildFormTemplate } from "./builder";
+
+export function LDCETemplate(passport: { data: object }) {
+  const get = (path: string): string => {
+    return _getString(passport.data, path);
+  };
+  const getBoolean = (path: string): boolean => {
+    return _getBoolean(passport.data, path);
+  };
+
+  return buildFormTemplate({
+    presets: {
+      title: "Application for a Lawful Development Certificate - Existing",
+      subtitle:
+        "Town and Country Planning Act 1990: Section 191 as amended by section 10 of the Planning and Compensation Act 1991. Town and Country Planning (Development Management Procedure) (England) Order 2015",
+    },
+    sections: [
+      {
+        title: "1. Applicant Name and Address",
+        fields: [
+          {
+            name: "Name",
+            value: `${get("applicant.name.first")} ${get(
+              "applicant.name.last"
+            )}`,
+          },
+          {
+            name: "Address",
+            value: get("applicant.address.singleLine"),
+          },
+        ],
+      },
+      {
+        title: "2. Agent Name and Address",
+        fields: [
+          {
+            name: "Is there an agent?",
+            value: getBoolean("applicant.agent.exists") ? "Yes" : "No",
+          },
+          {
+            name: "Agent name",
+            value: `${get("applicant.agent.name.first")} ${get(
+              "applicant.agent.name.last"
+            )}`,
+          },
+          {
+            name: "Agent address",
+            value: get("applicant.agent.address.singleLine"),
+          },
+        ],
+      },
+      {
+        title: "3. Site Address details",
+        fields: [
+          {
+            name: "Is the applicant’s address the same as the site address?",
+            value: get("applicant.address.sameAsSiteAddress"),
+          },
+          {
+            name: "Site address",
+            value: get("property.address.singleLine"),
+          },
+        ],
+      },
+      {
+        title: "4. Pre-application advice",
+        fields: [
+          {
+            name: "Has assistance or prior advice been sought from the local authority about this application?",
+            value: "Yes",
```

Comment:
There is a question and associated passport value for this - `application.preAppAdvice`